### ISSUE TITLE
The API server doesn't have a cloud provider flag

### ISSUE THREAD

**Issue by shannonxtreme** [kubernetes/website#46726]
Our cloud-controller-manager documentation is pages and blogs mention the following components that need to be configured for external cloud providers:

- kube-apiserver
- kube-controller-manager
- kubelet

kube-apiserver doesn't actually have a `cloud-provider` argument or configuration option in the API. 

Investigate which documentation is wrong. Do we need to remove mentions of kube-apiserver from the cloud-controller-manager docs, or do we need to add a missing cloud-provider option to the kube-apiserver docs? 

I can work on this because I'm already working on a user guide for cloud-provider-gcp

/sig docs
/sig cloud-provider
/language en
/assign

**Comment by tengqm** [comment]
If you run `kubelet --help` on v1.30, you can still see the `--cloud-provider` option.

**Comment by shannonxtreme** [comment]
> If you run `kubelet --help` on v1.30, you can still see the `--cloud-provider` option.
> 

kubelet uses it for sure but our cloud controller manager docs and blogs also say that kube-apiserver uses it, even though it doesn't seem to have that option documented elsewhere

**Comment by tengqm** [comment]
> > If you run `kubelet --help` on v1.30, you can still see the `--cloud-provider` option.
> 
> kubelet uses it for sure but our cloud controller manager docs and blogs also say that kube-apiserver uses it, even though it doesn't seem to have that option documented elsewhere

In that case, could you please be more specific which pages need fixes?
My take is that this issue won't be a difficult one to fix.

**Comment by shannonxtreme** [comment]
Yeah I don't think it'll be difficult either, but I wanted to make sure that we were fixing the correct problem before I tried to turn this into a help-wanted. I'll write out what needs to change at some point this week.

Do you know if we're allowed to change old blogs?

**Comment by tengqm** [comment]
We usually don't change blogs that are elder than one year.
